README.md

```diff
@@ -64,6 +64,8 @@ Alternatively, [tools/maptools.py](tools/README.md) can be used to build and ins
 
 Note - to build and run with Docker, see provided [building with docker](tools/docker/README.md)
 
+Builds for the master branch are also available through the following link: https://ftp.essensium.com/owncloud/index.php/s/6ob8ka1zYORDOVA .
```

Comment:
I think this does not belong in this README - should be in the WiKi instead IMO.

Reply:
I would think it is very normal for a README to specify where you can download binaries.

Reply:
OK, I actually used it now in another script so it is important that it is documented in the repository.